core/src/test/scala/org/apache/spark/deploy/SparkSubmitSuite.scala

```diff
@@ -504,6 +504,25 @@ class SparkSubmitSuite
     }
   }
 
+  test("SPARK-47475: Not to add primary resource to jars again" +
```

Comment:
Oh, this JIRA ID is wrong. We need to have SPARK-47495 like the PR title.

Reply:
Good catch!!!
Thanks for fixing this!